### PR TITLE
Fixes some input issues [no gbp]

### LINKED ISSUE #80516: Autolathe and crafting UI is bugged
## Issue Summary
Autolathe and the crafting searchbar is broken, and the autolathe fails to display anything searched up. 

## Round ID:
[220955](https://scrubby.melonmesa.com/round/220955)

## Reproduction:
Attempt to use the autolathe or the crafting UI's searchbar, the autolathe one refuses to work and the crafting one requires you to move the scroll bar in order for it to function.

### LINKED ISSUE #80528: TGUI searchbars jank
Reporting client version: 515.1623

## Issue Summary

the searchbar when you use a sheet inhand does not change results like it used to, only after it loses focus
protolathe searchbar just doesnt work at all
## Round ID:
[220994](https://scrubby.melonmesa.com/round/220994)

## Testmerges:
- [Save json savefiles to disk in pretty print form](https://www.github.com/tgstation/tgstation/pull/80512)
- [(Testmerge Ready) New Antagonist: Tiger Cooperative Fanatic](https://www.github.com/tgstation/tgstation/pull/80313)
- [[TM ONLY] Remove 'natural' beheading](https://www.github.com/tgstation/tgstation/pull/80439)
- [Tones down cyborg lover to cyborg fan](https://www.github.com/tgstation/tgstation/pull/80508)

## Reproduction:

1. use searchbar from the sheet build menu to search for a thing
2. results dont change
3. unfocus the searchbar by enter or something
4. results change

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Reported in discord: camera console filter needs to press enter
issue 2: designs not showing on search (this was a radar suggestion :| )
## Why It's Good For The Game
Fixes #80516
Fixes #80528
## Changelog
:cl:
fix: Camera console search should update automatically
fix: Autolathe search should now properly show designs
/:cl:

### CODE REVIEW

**Comment by SyncIt21** [comment]
Can #80528 also be fixed here? Seems simple enough

**Comment by RikuTheKiller** [comment]
> Can #80528 also be fixed here? Seems simple enough

they likely share the same issue so this will likely fix it

**Comment by jlsnow301** [comment]
> Can #80528 also be fixed here? Seems simple enough

yes it can!

File: tgui/packages/tgui/interfaces/StackCrafting.tsx [hunk 1]
```diff
@@ -57,7 +57,6 @@ type RecipeListFilterableEntry = [string, RecipeList | Recipe | undefined];
  * @param value the value to test
  * @returns type guard boolean
  */
-// eslint-disable-next-line func-style
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/StackCrafting.tsx:60]: no longer needed 😎

**Pull request merged**